runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -858,6 +858,122 @@ J9::Z::TreeEvaluator::pdclearSetSignEvaluator(TR::Node *node, TR::CodeGenerator
    return TR::TreeEvaluator::pdclearEvaluator(node, cg);
    }
 
+/*
+ * This method inlines the Java API StringCoding.hasNegatives(byte src, int off, int len) using
+ * SIMD instructions (it will only be invoked on supported processors).
```

Comment:
I think it should be ok if we remove the statement in the braces. (`it will only...`)

Reply:
Comment removed in f06ed6e41ee